src/streams/rtx_cache.rs

```diff
@@ -133,8 +135,8 @@ impl RtxCache {
 
     pub(crate) fn last_packet(&self) -> Option<&[u8]> {
         self.packet_by_seq_no
```

Comment:
`self.packet_by_seq_no.iter().rev().next().map(|e| e.1.payload.as_ref())` and you can avoid the new field

Reply:
Good point. Let's do that.